official/utils/logging/logger.py

```diff
@@ -149,6 +142,10 @@ def _collect_cpu_info(run_info):
 
   cpu_info["num_cores"] = multiprocessing.cpu_count()
 
+  # Note: cpuinfo is not installed in the TensorFlow OSS tree.
+  # It is installable via pip.
+  import cpuinfo
+
```

Comment:
You will probably need a `  # pylint: disable=g-import-not-at-top` to make glint clear these.

Reply:
Done.